internal/controllers/synthesis/backoff_test.go

```diff
@@ -39,60 +37,10 @@ func TestControllerBackoff(t *testing.T) {
 	t.Run("initial creation", func(t *testing.T) {
 		testutil.Eventually(t, func() bool {
 			require.NoError(t, client.IgnoreNotFound(cli.Get(ctx, client.ObjectKeyFromObject(comp), comp)))
-			return comp.Status.CurrentSynthesis != nil && comp.Status.CurrentSynthesis.Attempts >= 3
+			return comp.Status.CurrentSynthesis != nil && comp.Status.CurrentSynthesis.Attempts >= 10
 		})
 
-		// It shouldn't be possible to try three times within 250ms
+		// It shouldn't be possible to try this many times within 250ms
 		assert.Greater(t, int(time.Since(start).Milliseconds()), 250)
 	})
 }
-
-func TestControllerBackoffReset(t *testing.T) {
```

Comment:
This is no longer useful since the reset mechanism is very obvious, and integration testing it would be cumbersome.